spec/app/domain/authentication/authn-jwt/validate_status_spec.rb

```diff
@@ -73,6 +75,10 @@
       receive(:call).and_raise(fetch_issuer_configuration_is_invalid_error)
     )
 
+    allow(mocked_invalid_fetch_audience_value).to(
```

Comment:
Is there a usage for the mock?

Reply:
good catch, i created the infra without the test

Fixed :)